src/components/StyledButton.tsx

```diff
@@ -0,0 +1,48 @@
+import { View, StyleSheet } from 'react-native';
+import { Button } from 'react-native-elements';
+
+import globalStyles from '../styles/globalStyles';
+
+type StyledButtonProps = {
+  disabled: boolean;
+  text: string;
+  onPress: () => void;
+  extraStyles?: object;
+};
+
+function StyledButton({
+  disabled,
+  onPress,
+  text,
+  extraStyles = {},
+}: StyledButtonProps) {
+  return (
+    <View style={[styles.verticallySpaced, globalStyles.mt20, extraStyles]}>
+      <Button
+        disabledStyle={{
+          opacity: 0.4,
```

Comment:
This might be an android specific issue, but it should be changed nonetheless: the opacity is remaining at 100% for the button's background color. Please add `gwnOrangeDisabled: 'rgba(235, 86, 59, 0.4)'` to colors.ts and reference this color instead of manually setting opacity (which doesn't seem to be working)